app/models/sitemap_builder.rb

```diff
@@ -0,0 +1,35 @@
+class SitemapBuilder
+  def first_or_generate
+    Sitemap.first || Sitemap.create(xml: generate_xml(page_slugs + sign_slugs))
+  end
+
+  def update_sitemap
+    Sitemap.first.update(xml: generate_xml(page_slugs + sign_slugs))
```

Comment:
🐘 since you will always be calling for both page and sign slugs, and never individually, it seems worthy of being a method to me... even though the method is currently only combining 2 arrays.

Reply:
I've just changed this in the first or generate method above, do you still reckon itd be worth combining these two in a wrapper method or something though?

Reply:
It seems nice to have a method to call and know you will get ALL pages, but very much an 🐘, not important.